frontend/src/pages/CourseEditor/DefinitionStep/DefinitionStep.tsx

```diff
@@ -35,16 +42,23 @@ const DefinitionStep: FC<IStepProps> = ({
       <FormWrapper>
         <SectionName>{EditorTitles.definitionStepTitle}</SectionName>
         <SectionWrapper>
-          <Field
-            variant="outlined"
-            id="title"
-            name="title"
-            value={formik.values.title}
-            onChange={formik.handleChange}
-            onBlur={onFieldBlur}
-            error={Boolean(formik.errors?.title)}
-            helperText={formik.errors?.title}
-          />
+          <FieldWrapper>
+            <Field
+              variant="outlined"
+              id="title"
+              name="title"
+              label="Title"
+              value={formik.values.title}
+              onChange={formik.handleChange}
+              onBlur={onFieldBlur}
+              inputProps={{
+                maxLength: 100,
```

Comment:
Maybe it will be better to move in common variable to handle all the restrictions from one place?